{"name": "账号密码错误登录", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: './TestData/main.yml'", "trace": "self = <zikaow.TestCase.test_login.TestLogin object at 0x000002E050D31370>\n\n    def setup(self):  # 方法运行前执行一次\n>       self.loginPage = self.testDriver.go_my_login()\n\ntest_login.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\Page\\main.py:13: in go_my_login\n    self.steps('./TestData/main.yml', 'go_my_login')\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <zikaow.Page.main.Main object at 0x000002E04FFD4460>\npath = './TestData/main.yml', key = 'go_my_login', kwargs = {}\n\n    def steps(self, path, key, **kwargs):  # 定义操作步骤的方法，用于通过编写配置文件，执行相关用例\n>       with open(path, 'r', encoding='utf-8') as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: './TestData/main.yml'\n\n..\\Page\\basepage.py:114: FileNotFoundError"}, "parameters": [{"name": "user", "value": "''"}, {"name": "pw", "value": "''"}, {"name": "msg", "value": "'请输入账号'"}], "start": 1622737790489, "stop": 1622737790489, "uuid": "0bdc7389-612b-40c0-878f-8bfdf8ed4397", "historyId": "e5414b6883fac091e9e66e2b99dfad9d", "testCaseId": "351efa5f97d628365da09acf6a5e9ba8", "fullName": "TestCase.test_login.TestLogin#test_log_error", "labels": [{"name": "feature", "value": "用户登录模块"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-TCP89B8"}, {"name": "thread", "value": "10156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_login"}]}